src/handlers.js

```diff
@@ -245,6 +257,9 @@ async function jobHandler(message) {
   let organization = message.payload.organization;
   let repository = message.payload.repository;
   let sha = message.payload.details['event.head.sha'];
+  //console.log('..message passed to jobHandler', message);
+  let number = message.payload.details['event.pullNumber'];
```

Comment:
Let's call this `pullNumber` instead of just `number` to make it more obvious what it is!